manager/apis/app/v1beta1/fybrikapplication_webhook.go

```diff
@@ -14,7 +14,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 
 	"fybrik.io/fybrik/pkg/environment"
-	validate "fybrik.io/fybrik/pkg/taxonomy/validate"
+	validate "fybrik.io/fybrik/pkg/validate"
```

Comment:
the import alias is redundant (it has the same name as the package) you can remove it, just leave 
"fybrik.io/fybrik/pkg/validate"

Reply:
done